review/tests.py

```diff
@@ -51,3 +51,21 @@ def test_delete_related_dogowner(self, test_review):
         dogowner.delete()
         assert dogowner not in DogOwner.objects.all()
         assert test_review in Review.objects.all()
+
+    def test_get_review_by_daycare_id(self):
+        daycare = DayCare.create(email="review@gmail.com", username="review", password="pass", name="reviewDayCare",
+                                 description="This is test daycare for review", price_per_day=10, capacity=50,
+                                 area="Merkaz", city="Tel-Aviv", address="The best review")
+        reviews = Review.get_review_by_daycare_id(daycare.id)
+        assert len(reviews) == 0
```

Comment:
The function returns None in case there are no reviews, so that's what you should expect here. 
Any reason you check for len?

Reply:
The function return none in case there is ObjectDoesNotExist exception, this search will return empty QuerySet
<img width="652" alt="image" src="https://user-images.githubusercontent.com/13609044/163774411-67169345-c7ed-4e1a-b439-4a627692716e.png">

Reply:
Gotcha